CHANGELOG.asciidoc

```diff
@@ -20,6 +20,7 @@ https://github.com/elastic/apm-server/compare/71df0d96445df35afe27f38bcf734a0828
 - Updated systemd doc url {pull}354[354]
 - Updated readme doc urls {pull}356[356]
 - Use updated stack trace frame values for calculating error `grouping_keys` {pull}485[485] 
+- Change logic for setting function name after applying sourcemap {pull}502[502]
```

Comment:
Please add the PR number to the line describing the source map feature, like so:

`- Support for sourcemap mapping on incoming frontend requests {pull}381[381] {pull}502[502]`

instead of the line added here.

This is how i think about the changelog in general: The changelog should explain the difference between the versions. If a bug was not released in a version, we should not have a bug-fix for it in the changelog. I'll bring it up on the next APM Server meeting to discuss.

Reply:
makes sense, will change!